runtimes/go/versions/latest/src/go.mod

```diff
@@ -4,5 +4,5 @@ go 1.22.5
 
 replace openruntimes/handler v0.0.0 => /usr/local/build
 
-require openruntimes/handler v0.0.0
-require github.com/open-runtimes/types-for-go/v4 v4.0.5
+// require openruntimes/handler v0.0.0
```

Comment:
Remove?!? Seems unneeded?!? This is so good, it was always hussle with this when devleoping